root/components/common-macros.tt

```diff
@@ -1097,3 +1103,9 @@ note-[%- edit.id -%]-[%- loop.count() -%]
     </div>
   </div>
 [%~ END ~%]
+
+[%~ MACRO bugtracker_url(description) BLOCK;
```

Comment:
I made a macro (two, actually, there is also a JavaScript implementation below) to have less duplication.